devbox.lock

```diff
@@ -0,0 +1,20 @@
+{
+  "lockfile_version": "1",
+  "packages": {
+    "nodePackages.eslint@8.41.0": {
```

Comment:
Is the lockfile up-to-date with `devbox.json`?

It looks like we have `nodePackages.eslint` referenced here, but it's not in the `packages` array (and probably doesn't need to be, based on our Slack discussion).

Reply:
fixed - also went ahead and added the linter for SDK consistency.